chapters/traversal.md

```diff
@@ -1,83 +1,89 @@
 # Understand Jina Recursive Document Representation
 
-In Jina, each Document is represented as a recursive representation (tree).
+This guide explains how Documents are represented within the Jina framework. It is especially relevant for understanding how segmenters, crafters and rankers work.
 
-A rooted recursive representation has a root node and every node has X children.
-In Jina, the root node is the document itself, while the *left* & *right* children are referred to as *chunks* and *matches* respectively.
+## General overview
+
+In Jina, the term Document refers to more than just a simple dataset. A Document is used for both indexing and querying. A search query is also a Document itself.
+
+When you index a Document in Jina, it can be represented as a rooted recursive representation (tree). The representation consists of a *root node* and several *child nodes*. In Jina, the root node is the original Document itself, while the *left* and *right* children are referred to as *chunks* and *matches* respectively. The figure below illustrates a basic Document structure that consists of a Document (root node) as well as two child nodes (chunks and matches).
 
 ![rooted-binary-tree](./images/overview.png)
 
-The above image illustrates a basic document structure: A document (root node) and the two possible child nodes: *chunks* and *matches*.
-chunks are a sequence of documents which is attached to the root document with a higher `granularity` degree. `matches` is a sequence of documents which are semantically related to the root document.
-We'll dive into these concepts in this chapter:
+The two terms chunks and sequence are still a bit unclear. In short, chunks are a sequence of Documents which can be attached to any Document with a higher `granularity` degree. In the figure above this is the root node. 
```

Comment:
Change `The two terms chunks and sequence are still a bit unclear` to `Let's deep dive into the meaning of the two terms chunks and sequence`.

Change `attached to any Document` to `attached to parent Document`

Reply:
I have changed it into `require further explanation`, and added `parent` as you suggested